torch/testing/_internal/common_optimizers.py

```diff
@@ -119,7 +120,7 @@ def __init__(
         ),
         # A subset of the global-cliquey flags (fused, foreach, differentiable) the optimizer
         # supports. See NOTE: [optimizer kwarg categories] for what global-cliquey means.
-        supported_impls: Tuple[str] = ("foreach", "differentiable"),
+        supported_impls: Tuple[str, ...] = ("foreach", "differentiable"),
```

Comment:
enable typing for this file to make sure we don't regress these?

Reply:
Maybe later, the entire torch/testing/_internal directory is excluded from typing and it isn't obvious how to just bring this one file back with `pytorch/mypy.ini`